src/devices/cellular.c

```diff
@@ -404,7 +406,7 @@ static bool auth_telem_stream(struct serial_buffer *sb,
         return false;
 }
 
-int cellular_init_connection(DeviceConfig *config)
+int cellular_init_connection(DeviceConfig *config, millis_t * connected_at)
```

Comment:
Just a thought... should we pass a struct here so we can add more elements in the future if needed?

Reply:
Good idea. I think if we add a 3rd parameter we can do it at that time.